src/com/facebook/buck/jvm/kotlin/KotlincStep.java

```diff
@@ -168,13 +159,8 @@ public String getDescription(ExecutionContext context) {
       }
     }
 
-    if (context.getVerbosity().shouldUseVerbosityFlagIfAvailable()
-        && !verboseModeOnlyExtraArguments.isEmpty()) {
-      for (String extraArgument : verboseModeOnlyExtraArguments) {
-        if (!extraArgument.isEmpty()) {
-          builder.add(extraArgument);
-        }
-      }
+    if (context.getVerbosity().shouldUseVerbosityFlagIfAvailable()) {
```

Comment:
Kotlin verbosity will outputted if either it's set via buck's Context, or if it's set in the Kotlin config. Felt a bit odd, but better than alternatives (not providing kotlin config version, ignoring buck verbosity, or making both required)